scripts/microbe_stage/organelle.as

```diff
@@ -730,10 +743,13 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
             // Also add our offset to the hex offset
             Float3 translation = Hex::axialToCartesian(hex.q, hex.r) + this.cartesianPosition;
 
+            if(species.isBacteria)
+                translation /= 2;
```

Comment:
The cartesian position was already divided by two and here the result where it is added is divided by two again.